about-me.md

```diff
@@ -1,9 +1,9 @@
 # About Me
 
-![Picture of me]()
+![Picture of me](https://avatars0.githubusercontent.com/u/35268641?s=460&v=4)
 
-Hi, I'm ____________. I'm a ________ at *__________*.
+Hi, I'm Neil. I'm an Engineer at *OPM*.
```

Comment:
I think it's probably better for you to use *USDS*.

Reply:
ah you're right! Thanks :)